UOP1_Project/Packages/manifest.json

```diff
@@ -43,6 +43,7 @@
     "com.unity.modules.video": "1.0.0",
     "com.unity.modules.vr": "1.0.0",
     "com.unity.modules.wind": "1.0.0",
-    "com.unity.modules.xr": "1.0.0"
+    "com.unity.modules.xr": "1.0.0",
+	"com.bovinelabs.analyzers": "https://github.com/tertle/com.bovinelabs.analyzers.git"
```

Comment:
Didn't I read in the conversation that we were going to remove the dependency on this package?

Reply:
Yes, you did, but it turned out that when I restart VS the erros from StyleCop disappear, so we really need this package, but it is not something that we have to be afraid of.

Reply:
There should be something to do the static code analysis for us, right?

Reply:
It seems like Visual Studio should be able to handle the .editorconfig without any extra intervention

https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2019

Same with Rider 

https://www.jetbrains.com/help/rider/Using_EditorConfig.html#export-code-style-settings

Whilst I agree that we should have static code analysis, we shouldn't have platform/IDE specific ones and leave the rest up to chance as that would still allow for discrepancies to go through without anyone really noticing. Because of this I don't think it's a good idea to include that analyzer as it only seems to support Visual Studio and Rider, VSCode, WebStorms, Eclipse and many others are left out whilst it's highly likely that people will be using any of those IDEs.

In my opinion we should stick with just what's possible through the .editorconfig that the Github Action bot can handle as well, if we don't then there will still be inconsistencies

Reply:
@StanislavChankov Could you remove the usage of this package (both here and in the lockfile) and the subsequent stylecop.json if it was created by this?